README.md

```diff
@@ -0,0 +1,32 @@
+# OpenAstronomy conda channel
+
+## Installing packages
+
+To add this channel to your list of default conda channels:
+`conda config --add channels openastronomy`
+
+To install a single package from this channel:
+`conda install -c openastronomy <package>`
+
+## List of packages
+
+The list of packages on this channel is at
+https://anaconda.org/openastronomy/packages
+
+It includes all of the packages on these conda channels:
+
++ `astropy`: channel for [astropy](http://astropy.org) [affiliated packages](http://astropy.org/affiliated)
+
+## How can I add my package?
```

Comment:
Not strictly related to the README itself... but possibly.  @mwcraig : does this *automatically* include the affiliated package channel?  If so, perhaps this section should also include something along the lines of "if your package is or might be an astropy affiliated package, use that channel instead"?